minigui/winrate_graph.ts

```diff
@@ -45,10 +45,11 @@ class WinrateGraph extends View {
   protected mainLine: number[] = [];
   protected variation: number[] = [];
 
-  protected moveNum = 0;
-
   protected xScale = MIN_POINTS;
```

Comment:
is this xScale unneeded, now?

Reply:
added a comment describing why xScale is needed